date_range/static/src/js/date_range.js

```diff
@@ -16,6 +16,12 @@ odoo.define('date_range.search_filters', function (require) {
             this.is_date = field.type === 'date' || field.type === 'datetime';
             this.$value = this.$el.find('.searchview_extended_prop_value, .o_searchview_extended_prop_value');
             if (this.is_date) {
+//                this._rpc({
+//                    model: 'date.range.type',
+//                    method: 'search_read',
+//                    fields: ['name'],
+//                    context: this.context,
+//                    }).done(this.proxy('add_date_range_types_operator'));
```

Comment:
@mreficent This code must be removed id it's no more needed

Reply:
I added this code expressly to indicate what should be the way to fix the javascript error, this code should replace the next two lines. I put commented instead because it's not correct yet.

Reply:
@mreficent Is this PR therefore still a WIP?

Reply:
It's stated in https://github.com/OCA/server-ux/pull/95#issuecomment-537240167.